libs/net/entities/Content.cs

```diff
@@ -252,9 +252,9 @@ private Content(string uid, string headline, ContentType contentType, License li
         this.Uid = uid ?? throw new ArgumentNullException(nameof(uid));
         this.Headline = headline;
         this.ContentType = contentType;
-        this.LicenseId = license?.Id ?? throw new ArgumentNullException(nameof(license));
+        if (license != null) this.LicenseId = license.Id;
```

Comment:
You can't do this `LicenseId` is a required property.  It cannot default to `0`.

Reply:
The same comment as above, thanks